src/main/java/club/issizler/okyanus/api/entity/Player.java

```diff
@@ -18,7 +16,7 @@
      * @param returnFluids Should we return fluids?
      * @return Block, or empty if we couldn't find anything or we found an entity
      */
-    Optional<Block> getTargetBlock(double distance, boolean returnFluids);
+    Block getTargetBlock(double distance, boolean returnFluids);
```

Comment:
This breaks the API. Also please make this into another PR if you are planning to remove these optionals.